files/install_workflow.py

```diff
@@ -17,8 +18,13 @@ def main():
                         help="Galaxy admin user API key (required if not "
                              "defined in the tools list file)",)
     args = parser.parse_args()
+
     gi = galaxy.GalaxyInstance(url=args.galaxy_url, key=args.api_key)
-    gi.workflows.import_workflow_from_local_path(args.workflow_path)
+
+    import_uuid = json.load(open(args.workflow_path, 'r'))['uuid']
+    existing_uuids = [d['latest_workflow_uuid'] for d in gi.workflows.get_workflows()]
```

Comment:
Do old workflows also have `latest_workflow_uuid` key? I don't remember when this field was added to Galaxy but perhaps using `.get` here would be more resilient. (If someone knows how far back the `uuid` field dates and BioBlend is not even being tested with versions of Galaxy that old, I guess this comment can be ignored)